[2024-05-20T12:44:12.797+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_data_attendance_test.delete_att_data_into_db manual__2024-05-20T12:44:09.611406+00:00 [queued]>
[2024-05-20T12:44:12.803+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_data_attendance_test.delete_att_data_into_db manual__2024-05-20T12:44:09.611406+00:00 [queued]>
[2024-05-20T12:44:12.804+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2024-05-20T12:44:12.812+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): delete_att_data_into_db> on 2024-05-20 12:44:09.611406+00:00
[2024-05-20T12:44:12.816+0000] {standard_task_runner.py:60} INFO - Started process 2802 to run task
[2024-05-20T12:44:12.819+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'api_data_attendance_test', 'delete_att_data_into_db', 'manual__2024-05-20T12:44:09.611406+00:00', '--job-id', '831', '--raw', '--subdir', 'DAGS_FOLDER/api_data_attendance.py', '--cfg-path', '/tmp/tmp7tgrlwd4']
[2024-05-20T12:44:12.821+0000] {standard_task_runner.py:88} INFO - Job 831: Subtask delete_att_data_into_db
[2024-05-20T12:44:12.898+0000] {task_command.py:423} INFO - Running <TaskInstance: api_data_attendance_test.delete_att_data_into_db manual__2024-05-20T12:44:09.611406+00:00 [running]> on host e3e2dc6bc2be
[2024-05-20T12:44:12.946+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_data_attendance_test' AIRFLOW_CTX_TASK_ID='delete_att_data_into_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-20T12:44:09.611406+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-20T12:44:09.611406+00:00'
[2024-05-20T12:44:12.955+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-05-20T12:44:12.973+0000] {base.py:83} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-05-20T12:44:12.985+0000] {sql.py:450} INFO - Running statement: 
        DELETE FROM attendance;
        , parameters: None
[2024-05-20T12:44:12.992+0000] {sql.py:459} INFO - Rows affected: 0
[2024-05-20T12:44:12.996+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-05-20T12:44:13.005+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=api_data_attendance_test, task_id=delete_att_data_into_db, execution_date=20240520T124409, start_date=20240520T124412, end_date=20240520T124413
[2024-05-20T12:44:13.036+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-20T12:44:13.053+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
